pkgs/build-support/fetchurl/default.nix

```diff
@@ -124,6 +124,8 @@ if (!hasHash) then throw "Specify hash for fetchurl fixed-output derivation: ${s
 
   inherit curlOpts showURLs mirrorsFile impureEnvVars postFetch downloadToTemp executable;
 
+  userAgent = "Nix ${builtins.nixVersion} on nixpkgs ${stdenv.lib.nixpkgsVersion}";
```

Comment:
I think it would be enough to put nixos version in there, e.g. `16.09` ATM, or are there any expectations about the version?